utils/releaser/src/ReleaseWorker/Release/CreateAndCommitLockFilesReleaseWorker.php

```diff
@@ -33,15 +33,19 @@ public function getPriority(): int
      */
     public function work(Version $version): void
     {
-        $this->symfonyStyle->note('Removing vendor/, node_modules/, composer.lock, and package-lock.json');
+        $this->symfonyStyle->note('Removing vendor/, node_modules/, composer.lock, migrations-lock.yml, and package-lock.json');
         $this->processRunner->run('rm -rf project-base/vendor');
         $this->processRunner->run('rm -rf project-base/node_modules');
         $this->processRunner->run('rm -f project-base/composer.lock');
         $this->processRunner->run('rm -f project-base/package-lock.json');
+        $this->processRunner->run('rm -f project-base/migrations-lock.yml');
         $this->processRunner->run('cd project-base && composer install && npm install');
+        $this->processRunner->run('cd project-base && php phing db-rebuild');
```

Comment:
Why to rebuild?

Reply:
Because we want to drop the old database and create new, fresh one with only the migrations that you need so you can generate right `migrations-lock.yml`